how-to/how-to-display-animated-splash-screen.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to Display an Animated Splash Screen in a Hybrid App
+description: Make the splash screen with animated logo. Way to add an animated splash screen.
+type: how-to
+page_title: Show an Animated Splash Screen
+slug: display-animated-splash-screen
+position: 
+tags: appbuilder, cordova, coreplugins
+teampulseid:
+ticketid: 1066046, 1060903, 1057281
+pitsid:
+---
+                    
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code ({{site.ab}})</td>	
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Splash screen</td>	
+  </tr>
+  <tr>
+    <td>Plugin</td>
+    <td>Cordova Splash Screen</td>	
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid {{site.ac}}</td>	
+  </tr>
+</table>
+
+## Description
+
+With the integrated functionality for building hybrid mobile applications in {{site.tp}}, it is not possible to use an animated splash screen.
```

Comment:
Maybe switch the places of the two parts of the sentence. 
It is not possible to use an animated splash screen using the integrated functionality for building hybrid mobile applications in {{site.tp}}.